advanced/editor-command-identifiers.md

```diff
@@ -402,4 +424,92 @@ tinymce.execCommand('mceToggleEditor', false, '<editor_id>');
+| justifycenter        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `justifycenter` command.          |
+| justifyfull          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `justifyfull` command.            |
+| justifyleft          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `justifyleft` command.            |
+| justifyright         | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `justifyright` command.           |
+| mceblockquote        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `mceblockquote` command.          |
+| outdent              | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `outdent` command.                |
+| strikethrough        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `strikethrough` command.          |
+| subscript            | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `subscript` command.              |
+| superscript          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `superscript` command.            |
+| underline            | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `underline` command.              |
+
+**Examples**
+
+```js
+tinymce.activeEditor.queryCommandState('bold');
+tinymce.activeEditor.queryCommandState('insertdefinitionlist');
+tinymce.activeEditor.queryCommandState('insertorderedlist');
+tinymce.activeEditor.queryCommandState('insertunorderedlist');
+tinymce.activeEditor.queryCommandState('italic');
+tinymce.activeEditor.queryCommandState('justifycenter');
+tinymce.activeEditor.queryCommandState('justifyfull');
+tinymce.activeEditor.queryCommandState('justifyleft');
+tinymce.activeEditor.queryCommandState('justifyright');
+tinymce.activeEditor.queryCommandState('mceblockquote');
+tinymce.activeEditor.queryCommandState('outdent');
+tinymce.activeEditor.queryCommandState('strikethrough');
+tinymce.activeEditor.queryCommandState('subscript');
+tinymce.activeEditor.queryCommandState('superscript');
+tinymce.activeEditor.queryCommandState('underline');
+```
+
+## Queryable command values
+
+{{site.productname}} provides the `queryCommandValue` API to allow developers to determine the current state of selected content. The query will return an object containing the relevant value.
+
+### Listing core and plugin queryable command values
+
+To retrieve a list of avaliable queryable command values from the active editor, run the following command from the browser console:
+
+```js
+tinymce.activeEditor.editorCommands.commands.value;
+```
+
+### Available queryable command values
```

Comment:
Likewise the case for the command name here is incorrect.

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1590/commits/c4f6e844ae7168e5343fd3e281d776e5355b09ac